src/packages/end-to-end/src/__tests__/api/sqlite/query/export-page-size-decorator.test.ts

```diff
@@ -0,0 +1,147 @@
+import 'reflect-metadata';
+import gql from 'graphql-tag';
+import assert from 'assert';
+import Graphweaver from '@exogee/graphweaver-server';
+import { Entity, Property, PrimaryKey } from '@mikro-orm/core';
+import {
+	createBaseResolver,
+	Field,
+	GraphQLEntity,
+	ID,
+	ObjectType,
+	Resolver,
+	SummaryField,
+	ExportPageSize,
+} from '@exogee/graphweaver';
+import { BaseEntity, MikroBackendProvider } from '@exogee/graphweaver-mikroorm';
+import { Schema } from '@exogee/graphweaver-admin-ui-components';
+
+import { SqliteDriver } from '@mikro-orm/sqlite';
+
+/** Setup entities and resolvers  */
+@Entity({ tableName: 'Album' })
+class OrmAlbum extends BaseEntity {
+	@PrimaryKey({ fieldName: 'AlbumId', type: 'number' })
+	id!: number;
+
+	@Property({ fieldName: 'Title', type: 'NVARCHAR(160)' })
+	title!: unknown;
+}
+
+@Entity({ tableName: 'Artist' })
+class OrmArtist extends BaseEntity {
+	@PrimaryKey({ fieldName: 'ArtistId', type: 'number' })
+	id!: number;
+
+	@Property({ fieldName: 'Name', type: 'NVARCHAR(120)', nullable: true })
+	name?: unknown;
+}
+
+@ExportPageSize(500)
+@ObjectType('Album')
+export class Album extends GraphQLEntity<OrmAlbum> {
+	public dataEntity!: OrmAlbum;
+
+	@Field(() => ID)
+	id!: number;
+
+	@SummaryField()
+	@Field(() => String)
+	title!: string;
+}
+
+@ExportPageSize(100)
+@ObjectType('Artist')
+export class Artist extends GraphQLEntity<OrmArtist> {
+	public dataEntity!: OrmArtist;
+
+	@Field(() => ID)
+	id!: number;
+
+	@SummaryField()
+	@Field(() => String, { nullable: true })
+	name?: string;
+}
+
+const connection = {
+	connectionManagerId: 'sqlite',
+	mikroOrmConfig: {
+		entities: [OrmAlbum, OrmArtist],
+		driver: SqliteDriver,
+		dbName: 'databases/database.sqlite',
+	},
+};
+
+@Resolver((of) => Album)
+class AlbumResolver extends createBaseResolver<Album, OrmAlbum>(
+	Album,
+	new MikroBackendProvider(OrmAlbum, connection)
+) {}
+
+@Resolver((of) => Artist)
+class ArtistResolver extends createBaseResolver<Artist, OrmArtist>(
+	Artist,
+	new MikroBackendProvider(OrmArtist, connection)
+) {}
+
+test('Should return isReadOnly attribute for each entity in getAdminUiMetadata', async () => {
```

Comment:
I think this needs to describe the export page size

Reply:
fixed